test/ui/Driver.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    Builder,
+    Capabilities,
+    type ThenableWebDriver,
+} from 'selenium-webdriver'
+import { Options } from 'selenium-webdriver/chrome'
+
+export function BuildDriver(): ThenableWebDriver {
+    const options = new Options()
+
+    if (process.env.RUN_HEADLESS === 'true') {
```

Comment:
This is going to be slightly dodgy as TypeScript treats anything in `process.env` as a string, so you might want to change this to a variable that isn't a true/false value

Reply:
okay will update this now